packages/aws-cdk/lib/api/monitor/logs-monitor.ts

```diff
@@ -0,0 +1,196 @@
+import * as util from 'util';
```

Comment:
Maybe `lib/api/logs/` is a better directory for this file than `lib/api/monitor`?